ca_gazebo/worlds/april_tags.world

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<sdf version="1.6">
+  <world name="default">
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+      <pose> 0 0 50000 0 0 0</pose>
+    </include>
+
+    <!-- A ground plane -->
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+
+    <!--gravity> 0.0 0.0 0.0 </gravity-->
+
+    <physics type="ode">
+      <real_time_update_rate>1000.0</real_time_update_rate>
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>10</real_time_factor>
+      <ode>
+        <solver>
+          <type>quick</type>
+          <iters>150</iters>
+          <precon_iters>0</precon_iters>
+          <sor>1.400000</sor>
+          <use_dynamic_moi_rescaling>1</use_dynamic_moi_rescaling>
+        </solver>
+        <constraints>
+          <cfm>0.00001</cfm>
+          <erp>0.2</erp>
+          <contact_max_correcting_vel>2000.000000</contact_max_correcting_vel>
+          <contact_surface_layer>0.01000</contact_surface_layer>
+        </constraints>
+      </ode>
+    </physics>
+
+
+    <include>
+      <uri>model://create_four_walls</uri>
+    </include>
+
+    <light type="spot" name="light_1">
+      <pose>-10 7 1000 0 0 0</pose>
```

Comment:
Isn't this spot too high?

Reply:
Yes. In fact, the light aren't necessary at all in this case. I'll delete them.